### DISCUSSION TITLE
怎么解决啊，第一个链接还打不开。。

### DISCUSSION THREAD

**Discussion by jjismhg** [mikumifa/biliTickerBuy#160] [提问专区]
![Uploading 7c9048957789ff660a024e7fd0e0116.png…]()

**Comment by jjismhg** [comment]
<img width="876" alt="7c9048957789ff660a024e7fd0e0116" src="https://github.com/mikumifa/biliTickerBuy/assets/173658374/8d489bca-879f-4c4c-b574-d3f3079cb433">

**Reply by emm20060814** [reply]
请问后面怎么做？

**Reply by emm20060814** [reply]
什么叫exe启动。

**Reply by mikumifa** [reply]
看说明书

**Comment by mikumifa** [comment]
http://127.0.0.1:7860/

**Reply by slkduiers** [reply]
我试了一下您的这个网址，它还是拒绝连接请求，开关VPN都不行

**Reply by mikumifa** [reply]
你exe启动了吗

**Reply by slkduiers** [reply]
解决了，再次运行发现是我的防火墙把里面的一个文件杀掉了
